gallery/plot_transforms.py

```diff
@@ -62,3 +62,27 @@ def _plot(img, title, **kwargs):
 # converts an image to grayscale
 gray_img = T.Grayscale()(orig_img)
 plot(gray_img, "Grayscale image", cmap='gray')
+
+####################################
+# RandomPerspective
+# -----------------
+# The :class:`~torchvision.transforms.RandomPerspective` transform
```

Comment:
Here and below, let's add a "see also `the functional equivalent`" as done for the other transforms (except for the jitter because it doesn't exists).

Reply:
Thanks! I found the artifacts.
I'll add the non-random equivalents.